cogs/starboard.py

```diff
@@ -98,12 +100,13 @@ async def _build_embeds(self, msg: discord.Message) -> List[discord.Embed]:
         return [main_embed, reply_embed]
 
     async def create_starboard_post(self, react: discord.Reaction):
-        embeds = await self._build_embeds(react.message)
-        open_msg_view = await self._get_open_msg_view(react.message)
-        msg: discord.Message = await self.starboard_channel.send(
-            self._get_title(react), embeds=embeds, view=open_msg_view
-        )
-        self.starboard_msgs[react.message.id] = msg.id
+        async with self.__lock:
```

Comment:
Was this meant to be included in a separate PR? I'm not sure what this change has to do with this PR

Reply:
You are correct this should not be part of this PR. I'll fix it.